gulp/tasks/deploy.js

```diff
@@ -58,6 +59,9 @@ gulp.task('deploy:dist', ['sass:foundation', 'javascript:foundation'], function(
       .pipe(rename({ suffix: '.min' }))
       .pipe(gulp.dest('./dist/css'))
     .pipe(cssFilter.restore)
+    .pipe(sourcemapFilter)
+    .pipe(gulp.dest('./dist/css'))
```

Comment:
Missing tabulation here.

Reply:
Good catch, will change it tomorrow morning if this is ok.

Reply:
Changed.